<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="modelColorsTab">
        <attribute name="title">
         <string>Model Colors</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <widget class="QLabel" name="markedEdgeColorLabel">
           <property name="text">
            <string>Marked Edge Color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="markedFacetColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QToolButton" name="vertexColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QFrame" name="markedVertexColorFrame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="edgeColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QDoubleSpinBox" name="vertexSizeSpinBox"/>
         </item>
         <item row="7" column="0">
          <widget class="QDoubleSpinBox" name="edgeSizeSpinBox"/>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="markedEdgeColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QToolButton" name="markedVertexColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="edgeColorLabel">
           <property name="text">
            <string>Edge Color</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="facetColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="markedEdgeColorFrame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="markedFacetColorLabel">
           <property name="text">
            <string>Marked Facet Color</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="edgeSizeLabel">
           <property name="text">
            <string>Edge Size</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="vertexColorLabel">
           <property name="text">
            <string>Vertex Color</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="facetColorFrame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="vertexSizeLabel">
           <property name="text">
            <string>Vertex Size</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="facetColorLabel">
           <property name="text">
            <string>Facet Color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QFrame" name="markedFacetColorFrame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QFrame" name="edgeColorFrame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QFrame" name="vertexColorFrame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="markedVertexColorLabel">
           <property name="text">
            <string>Marked Vertex Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="outputOptions">
        <attribute name="title">
         <string>Output</string>
        </attribute>
        <widget class="QLabel" name="precisionLabel">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>141</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Output Precision</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="precisionSpinBox">
         <property name="geometry">
          <rect>
           <x>7</x>
           <y>10</y>
           <width>341</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>328</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Always save files before compile and render</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
